Source/WebGPU/WebGPU/ComputePipeline.h

```diff
@@ -50,9 +50,9 @@ class ComputePipeline : public WGPUComputePipelineImpl, public RefCounted<Comput
     {
         return adoptRef(*new ComputePipeline(computePipelineState, pipelineLayout, threadsPerThreadgroup, device));
     }
-    static Ref<ComputePipeline> create(id<MTLComputePipelineState> computePipelineState, MTLComputePipelineReflection* reflection, MTLSize threadsPerThreadgroup, Device& device)
+    static Ref<ComputePipeline> create(id<MTLComputePipelineState> computePipelineState, Ref<PipelineLayout> pipelineLayout, MTLSize threadsPerThreadgroup, Device& device)
```

Comment:
Excellent.

Should this be `Ref<PipelineLayout>&&`?

Reply:
Yes I will fix that, thank you for noticing